guides/source/configuring.md

```diff
@@ -692,7 +692,7 @@ Accepts a logger conforming to the interface of Log4r or the default Ruby `Logge
 
 #### `config.assets.quiet`
 
-Disables logging of assets requests. Set to `true` by default in `development.rb`.
+Disables logging of assets requests. Set to `true` by default in `config/environments/development.rb`.
```

Comment:
There is still one instance of `development.rb` without the full path at `config.assets.debug`. Can you update that as well?

Reply:
@MatheusRich See https://github.com/rails/rails/pull/52544